scoring.js

```diff
@@ -36,8 +36,28 @@ function calculateReturnScore(player) {
 }
 
 module.exports.calculateScore = function (player) {
-  return calculatePassingScore(player) +
-    calculateRushingScore(player) +
-    calculateReceivingScore(player) +
-    calculateReturnScore(player)
+  switch (player.position) {
+    case "QB":
+      return calculatePassingScore(player) +
+        calculateRushingScore(player)
+      break
```

Comment:
Oh, I bet this was your problem with ESLint. So these breaks are actually unreachable so that is what it would have been complaining about. In general, yes we want breaks in our switch statements so that we only run a single case. But remember we said that having a return will also jump you out of the switch statement in the same way. So since we have the return statement on the line above, that will always run and take us out of the switch and it would be impossible to ever get to the break statement. That's why ESLint called it unreachable code. We can safely delete the break statements and it will all work correctly because of the return statements.